job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -143,6 +143,11 @@ def final REPO_CONFIGS = [
                 artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
                         "kie-wb-tests/kie-wb-tests-gui/target/screenshots/**"
                 ],
+                downstreamRepos        : ["jbpm-work-items"]
+        ],
+        "jbpm-work-items"                  : [
+                label                  : "rhel7 && mem4g",
+                ircNotificationChannels: ["#jbpmdev"],
```

Comment:
I am afraid this is going to be a bit harder. The current jobs are building the needed upstream repos from sources, based on `branch-mapping.yaml` and `repository-list.txt`. The `jbpm-work-items` repo is not in any of those files, so the build will fail. I see two options here:

 1) rely on remote SNAPSHOTs (and don't build any upstream repos)
 2) "hack" the config and build the upstream repos up until jbpm (that should be enough)

I would probably vote for 2. as that will make the jobs a bit more stable (avoiding the remote SNAPSHOT dependency).

Reply:
@psiroky I don't know how to do this - I thought I am doing it right, but I see it was not.
Tbh I don't know what you mean with 2.
To prevent our endless loop of modifications until I do it it right I ask you please to do this.